### PR TITLE
Add tf.keras.layers.Log_Softmax

### PR DESCRIPTION
**Pull request by XuesongYang**
Applies the logarithm after softmax function of the logit. This implementation should be more efficient using 
logsoftmax = logits - log(reduce_sum(exp(logits), axis))

### CODE REVIEW

**Comment by caisq** [comment]
@XuesongYang Looks like this is a new layer type and it doesn't exist in keras-team/keras yet. Adding folks from the Keras team as reviewers.

**Comment by fchollet** [comment]
Thanks for the PR.

We don't add new layers that are simply wrapping a simple stateless TF op (in this case `nn.log_softmax_v2`). You can just use `nn.log_softmax_v2` as-is, or if you need to wrap it in a layer, you can use `tf.keras.layers.Lambda(nn.log_softmax_v2)`.

**Comment by Jun-depo** [comment]
I wish you can add this to Keras.  I had troubles with this.  I took me quite some time to land this page to learn this lambda wrap approach.  At least, I learned something new.
